### PR TITLE
Updating DOM and CSS for the mobile hero version.

### LINKED ISSUE #92: Improve display of kids list image for mobile

### PR DESCRIPTION
**Pull request by EdwinGuzman**
Fixes #92 

Don't want to override what's on dev so here are some screenshots. Pull the branch to check it out on mobile/desktop.

screen shot 2017-11-06 at 1 33 17 pm
screen shot 2017-11-06 at 1 33 08 pm

### CODE REVIEW

**Comment by ricardoom** [comment]
screen shot 2017-11-06 at 2 02 12 pm
small snag...

File: src/client/styles/components/_hero.scss [hunk 1]
```diff
@@ -1,61 +1,79 @@
 .hero {
-  background-color: #d0343a;
-  -webkit-background-size: cover;
-  -moz-background-size: cover;
-  background-size: cover;
-  background-position: center;
+  background-color: #fff;
   border-bottom: 0.125rem solid #eee;
```
Comment (ricardoom) [inline src/client/styles/components/_hero.scss]: .hero {

  @include min-screen($mobile-breakpoint) {
    border-bottom: 0.125rem solid #eee;  
 
 }

**Comment by EdwinGuzman** [comment]
@ricardoom Is that on a very wide viewport? or when do you see that?

**Comment by ricardoom** [comment]
@EdwinGuzman yeah, when the VP is very wide, we see how the layout breaks...

**Comment by EdwinGuzman** [comment]
@ricardoom okay, updated. Shouldn't go all the way to the left now.

**Comment by EdwinGuzman** [comment]
Check it http://staff-picks-dev.us-east-1.elasticbeanstalk.com/books-music-dvds/recommendations/best-books/ya

**Comment by ricardoom** [comment]
Same here. thanks!

**Pull request merged**